gis-data-sharing/open-data/index.md

```diff
@@ -6,16 +6,53 @@ author:
+  - NOTE: This will also allow people to export/copy some or all of the features if they add the dataset as a layer in ArcGIS Pro.
+
+## Taming the Sources List
+
+When a user clicks on a data category or searches for a dataset on the SGID Open Data site, a "sources" filter list appears on the search results page. Currently, **it can only show a max of ten sources**. Once we have more than ten in all our Open Data (not just the current search), we don't have any control over which ones are shown. In addition, the free-entry text box above the list requires a perfect text match, so it doesn't help users discover other sources either.
+
+We've tried to be careful with our data to keep any superfluous sources out of this list, and we'd appreciate your help with this as well. Please ensure you've only got one "source" value for your department or division.
+
+So where do the source values come from? Esri explains it [in this FAQ](https://doc.arcgis.com/en/hub/get-started/frequently-asked-questions.htm#GUID-9843B713-46D2-4938-A961-EC0CD81AE410). The main take away is that it first searches the **ArcGIS Online item's metadata** for an organization name. If it doesn't find anything there, it falls back to your **ArcGIS organization name**.
+
+If you'd like to customize the name displayed in the sources list, we suggest setting it in the `Resource > Citation > Contact > Organization Name` metadata field. Otherwise, make sure the metadata contact fields are empty and let it pick up your ArcGIS Online organization name. We use this second way to show "Utah Automated Geographic Reference Center (AGRC)" as the source for all our datasets.
+
+To change the organization name in the ArcGIS Online item's metadata, you need to make sure your ArcGIS Online organization has [metadata enabled](https://doc.arcgis.com/en/arcgis-online/administer/configure-details.htm#ESRI_SECTION1_7FE1F060E03046E692BB36E5F6E3B2AE). Then you can use the Metadata editor on the ArcGIS Online item page to set either field:
+
+![Citation-Contact]({% link images/od_citation_contact_orgname.png %}){: style="width:400px; margin: 10px" .outline } ![Contact]({% link images/od_contact_orgname.png %}){: style="width:400px; margin: 10px" .outline }
+
+Depending on how you upload or overwrite your ArcGIS Online data, it may also overwrite the item's metadata with info from the source **feature class metadata**. We suggest you make any changes in both the feature class and ArcGIS Online metadata to avoid problems in the futre. In the FGDC CSDGM style, the `Resource > Citation > Contact > Organization Name` field is in `Overview > Citation Contacts`. The `Resource > Contact > Organization Name` field is in `Resource > Points of Contact`.
```

Comment:
```suggestion
Depending on how you upload or overwrite your ArcGIS Online data, it may also overwrite the item's metadata with info from the source **feature class metadata**. We suggest you make any changes in both the feature class and ArcGIS Online metadata to avoid problems in the future. In the FGDC CSDGM style, the `Resource > Citation > Contact > Organization Name` field is in `Overview > Citation Contacts`. The `Resource > Contact > Organization Name` field is in `Resource > Points of Contact`.
```